static/js/login_or_signup.js

```diff
@@ -118,6 +123,12 @@ $(function() {
                 touchAllFields(formMetadata.$form, formMetadata.fields);
             }
         });
+
+        formMetadata.$form.find("input:password").keyup((event) => {
```

Comment:
you did `keyup` instead of `keypress`. Can you explain to me why? I forget the difference.

Reply:
actually reading up on the difference i think it should be `keydown`. The difference between the 3 (up, down and press) is weather or not the character for that key even is inserted into the "text" field: down is before, press is during and up is after.